### PR TITLE
Comments phrasing and typos

### PR DESCRIPTION
**Pull request by olekspickle**

### CODE REVIEW

File: migrate-core/src/lib.rs [hunk 1]
```diff
@@ -37,39 +37,38 @@ use std::fmt;
...
-    /// This method should cancel the changes made by the forward migration logic
-    /// and basically rollback the state of the migration object to the state
+    /// This method should cancel changes made by forward migration logic
+    /// and basically rollback the state of migration object to the state
     /// it was before [`Migration::up()`] was called.
     async fn down(&mut self, ctx: &mut Self::Ctx) -> Result<(), DynError>;
 }
 
-/// Bbuilder for [`Plan`] to allow its convenient configuration
+/// Builder for [`Plan`] to allow it's convenient configuration
```
Comment (Veetaha) [inline migrate-core/src/lib.rs]:
```suggestion
/// Builder for [`Plan`] to allow its convenient configuration
```
It's is short for `it is`, but here `its` is [correct](https://www.faa.gov/about/initiatives/plain_language/articles/its/)

File: migrate-core/src/lib.rs [hunk 2]
```diff
@@ -1,13 +1,13 @@
-//! The core crate that exposes APIs designed for full programmatic
+//! Core crate that exposes APIs designed for full programmatic
 //! configuration of migrations.
 //! See [`migrate`] crate if you need a CLI wrapper instead.
 //!
-//! The crate is built on top of [`migrate_state`] and the ideas outlined there.
+//! Crate is built on top of [`migrate_state`] and the ideas outlined there.
```
Comment (Veetaha) [inline migrate-core/src/lib.rs]:
```suggestion
//! The crate is built on top of [`migrate_state`] and the ideas outlined there.
```
`The` here is important to denote that it's the crate where this documentation is written in. I don't remember where, but I think I've heard the rule "if you can replace `the` with `this`, then it's correct"

**Review by Veetaha — APPROVED** [review]
Thanks for the contribution!

File: migrate-core/src/lib.rs (hunk 1 above)
Comment (olekspickle) [reply migrate-core/src/lib.rs]: aha, ok, I missed some context there

File: migrate-core/src/lib.rs (hunk 2 above)
Comment (olekspickle) [reply migrate-core/src/lib.rs]: I guess you are right

**Pull request merged**